packages/mobile/src/services/audio-player/NativeMobileAudio.ts

```diff
@@ -21,7 +21,4 @@ export class NativeMobileAudio {
   getAudioPlaybackRate = () => 1.0
   onBufferingChange = () => {}
   onError = () => {}
-  setPlaybackRate = () => {}
```

Comment:
ah i just removed these in my PR too haha

Reply:
omg lol, yeah i think i had an old branch that got merged 🗡️ were dancing around, lol